pyi.Dockerfile

```diff
@@ -13,7 +13,7 @@ ARG GPROFILER_BUILDER
 ARG PYPERF_BUILDER_UBUNTU
 ARG DOTNET_BUILDER
 ARG NODE_PACKAGE_BUILDER_MUSL
-ARG NODE_PACKAGE_BUILDER_GLIBC
+ARG NODE_PACKAGE_BUILDER_GLIBC=build-prepare
```

Comment:
Instead of specifying a default value here, I'd pass this value in the Aarch64 scripts (so that the x86_64 scripts don't have any interaction with it).

Reply:
I know it's different from the `Dockerfile` where the defaults are for x86_64, I think we should remove the defaults there as well :sweat_smile: and keep it consistent.